docs/configuration.md

```diff
@@ -29,6 +29,7 @@ initial-development: false  # OPTIONAL, defaults to `true`
 | `enable` | `None` | List of rules to enable as part of the checker (some rules are disabled by default) |
 | `allowed-branches` | `.*` | A regex specifying from which branch(es) releases and Git tags are allowed to be created |
 | `initial-development` | `true` | A boolean indicating that this project is still under _initial development_. During this state, any commit message containing a breaking change will result in a `MINOR` version bump. |
+| `sdkver-create-release-branches` | `false` | For SdkVer versioning scheme only: push a new branch if an RC or release build is performed on a non-release branch. If this config value is boolean `true`, the branch shall be of the form `release/N.N`. If this value is set to a string, it shall be used as the branch name prefix and appended with the major and minor release numbers, e.g. config value  `"rel/"` results in a branch named `rel/N.N`. |
```

Comment:
How would the configuration `release-branches` be handled in combination with this configuration item?